src/rtctools_heat_network/electricity_physics_mixin.py

```diff
@@ -488,17 +496,26 @@ def __electrolyzer_path_constaint(self, ensemble_member):
             constraints.extend(
                 [
                     (
-                        (gas_mass_flow_out_vect - gass_mass_out_linearized_vect) / nominal,
+                        (
```

Comment:
I think the docsting for this function should also be updated?

Reply:
updated